src/modules/report/services/kmsEncryption.ts

```diff
@@ -0,0 +1,57 @@
+import { KMSClient, EncryptCommand, DecryptCommand } from '@aws-sdk/client-kms'
+import { logger } from '../../../core/services/LoggerService'
+
+const kmsClient = new KMSClient({
+  region: process.env.AWS_REGION || 'us-east-1',
+})
+
+const kmsKeyId = process.env.AWS_KMS_KEY_ID
+
+const DATA_PREFIX = 'ENC_'
+
+export async function encryptData(data: string): Promise<string> {
+  if (!kmsKeyId) {
+    return data
```

Comment:
Could you add a comment that data continues to be stored as plain text when the AWS key is missing.

Reply:
added to README